google/google_container_cluster/separate_node_pool/main.tf

```diff
@@ -2,7 +2,7 @@
 
 # Documentation: https://www.terraform.io/docs/language/settings/index.html
 terraform {
-  required_version = ">= 0.14.0"
+  required_version = ">= 1.0.0"
```

Comment:
This results in a CI `terraform apply` error:
- https://github.com/ContainerSolutions/terraform-examples/runs/3281538654?check_suite_focus=true#step:5:4398

The error seems to bu unrelated to the terraform upgrade:
```
Plan: 2 to add, 0 to change, 0 to destroy.
google_container_cluster.changeme_separate_node_pool_cluster: Creating...
╷
│ Error: googleapi: Error 409: Already exists: projects/***/zones/us-central1-a/clusters/changeme-separate-node-pool-cluster., alreadyExists
│ 
│   with google_container_cluster.changeme_separate_node_pool_cluster,
│   on main.tf line 28, in resource "google_container_cluster" "changeme_separate_node_pool_cluster":
│   28: resource "google_container_cluster" "changeme_separate_node_pool_cluster" ***
│ 
╵
/home/runner/work/terraform-examples/terraform-examples/google/google_container_cluster/separate_node_pool
There are some problems with the CLI configuration:
╷
│ Error: Error parsing /home/runner/.terraformrc: At 3:16: illegal char
│
╵

As a result of the above problems, Terraform may not behave as intended.
```

Reply:
I think it can be due to some leaked resources in google cloud. Maybe something wasn't deleted in some previous runs.
@ianmiell I think we need to check this in the GCP console.